person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
     @id = Random.rand(1..1000)
     @name = name
```

Comment:
- [ ] When I try to test your application, I get this error. Kindly fix it

![image](https://user-images.githubusercontent.com/98453979/218804096-891e7c23-e2b7-4f7a-8552-26e17e2794d7.png)